mne/report/report.py

```diff
@@ -3218,7 +3218,9 @@ def _add_raw(
             init_kwargs, plot_kwargs = _split_psd_kwargs(kwargs=add_psd)
             init_kwargs.setdefault("fmax", fmax)
             plot_kwargs.setdefault("show", False)
-            fig = raw.compute_psd(**init_kwargs).plot(**plot_kwargs)
+            with warnings.catch_warnings():
+                warnings.simplefilter(action="ignore", category=FutureWarning)
+                fig = raw.compute_psd(**init_kwargs).plot(**plot_kwargs)
```

Comment:
any particular reason to use `catch_warnings`/`simplefilter` instead of `with pytest.warns(FutureWarning, match="...")`?

Reply:
Because it's not a test?

Reply:
🤦🏻 sorry